.github/ISSUE_TEMPLATE/ISSUE_TEMPLATE.yml

```diff
@@ -6,9 +6,9 @@ body:
     attributes:
       label: What is the issue?
       description: |
-       For UI, attach a screenshot.
+       For UI, attach a screenshot that <b>does not incude</b> unlicensed content.
```

Comment:
Hmm. This text is in Markdown format, not HTML, isn't it?

Reply:
YAML

Reply:
A little bit of misunderstanding here.
Yes, you are editing YAML file, you are right.
But the text you are editing will be used in GitHub issue template, which uses Markdown.

Reply:
Thanks @nevack